apps/cookbook/src/app/component-overview/component-configuration.json

```diff
@@ -176,19 +176,19 @@
     },
     {
       "title": "Spinner",
-      "paragraph": "The spinner indicates the loading state of a component or page.",
+      "paragraph": "A Spinner is a visual indicator that the for loading content or performing another process that the user needs to wait on.",
```

Comment:
This sentence sounds a bit funny.

Reply:
I agree but that's directly from the guidelines on Zeplin for the spinner component 😅

Reply:
Do you know who wrote the guidelines? I think you should ask them if they could rewrite the sentence :) I think it makes more sense if it's changed to something like:

"A Spinner is a visual indicator for loading content or performing another process that the user needs to wait on."

Reply:
I don't know who wrote it. But I'll ask Henrik :D and I agree that it should be changed